src/components/Cropper.js

```diff
@@ -233,29 +250,89 @@ export default class Cropper extends Component {
   }
 
   /**
-   * Increment current zoom by 0.5
+   * Retrieve min zoom based on hole size
+   * @todo refacto to component did update ?
+   */
+  _getMinZoom() {
+    const { zoomMin, holeSize } = this.props
+    const { pictureWidth } = this.state
+    let effectiveZoomMin = zoomMin
+    const maxPictureZoomOut = Math.ceil(holeSize / pictureWidth * 10) / 10
+    if (!zoomMin || zoomMin < maxPictureZoomOut) {
+      effectiveZoomMin = maxPictureZoomOut
+    }
+    return effectiveZoomMin
+  }
+
+  /**
+   * Calculate most appropriate step size
+   */
+  _getCurrentStepValue() {
+    const { zoomStep } = this.props
+    const { currentZoom } = this.state
+    if (currentZoom - zoomStep > 1) {
+      return zoomStep
+    } else if (currentZoom > this._getMinZoom() && (currentZoom - this._interpolateWithScale(0.1)) < this._getMinZoom()) {
+      return currentZoom - this._getMinZoom()
+    } else {
+      return 0.1
+    }
+  }
+
+  /**
+   * Check if passed zoom allow to zoom plus
+   * @param {Number} zoom
+   */
+  _canZoomIn(zoom) {
+    const { zoomMax } = this.props
```

Comment:
```suggestion
    const { maxZoom } = this.props
```
Plutot dans ce sens en anglais

Reply:
Cette props a été créé lors de la création de la lib, il faudrait créer une major pour ça :/